### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by Farah189** [javiribera/weighted-hausdorff-loss#3]
Hi, Thanks for providing the code. It is mentioned in the code that a pre-trained model comes with this package. The pre-trained model "unet_256x256_sorghum.ckpt" is not available in checkpoints folder. I was trying to test the code initially on the provided test set. It is giving an error on this file while testing. Should I train it again with any data set first then?

**Comment by Farah189** [comment]
Also xml file format is required for gt file having ground truth values. But the dataset_256x256 has gt.csv files. And training module is not accepting this format.
